roles/openshift_logging/README.md

```diff
@@ -199,3 +199,14 @@ Elasticsearch OPS too, if using an OPS cluster:
   Defaults to 'logging-mux'.
 - `openshift_logging_mux_file_buffer_storage_group`: The storage group used for Mux.
   Defaults to '65534'.
+
+### remote syslog forwarding
+`openshift_logging_fluentd_remote_syslog`: Set true to enable remote syslog forwarding, defaults to false
+`openshift_logging_fluentd_remote_syslog_host`: Required, hostname or IP of remote syslog server
+`openshift_logging_fluentd_remote_syslog_port`: Port of remote syslog server, defaults to 514
+`openshift_logging_fluentd_remote_syslog_severity`: Syslog severity level, defaults to debug
+`openshift_logging_fluentd_remote_syslog_facility`: Syslog facility, defaults to local0
+`openshift_logging_fluentd_remote_syslog_remove_tag_prefix`: Remove the prefix from the tag, defaults to ''
+`openshift_logging_fluentd_remote_syslog_tag_key`: If string specified, use this field from the record to set the key field on the syslog message
+`openshift_logging_fluentd_remote_syslog_use_record`: Set true to use the severity and facility from the record, defaults to false
+`openshift_logging_fluentd_remote_syslog_payload_key`: If string is specified, use this field from the record as the payload on the syslog message
```

Comment:
Add another line at the end like "There are corresponding `openshift_logging_mux_` parameters to the above for mux."